### PR TITLE
Snips

### PR DESCRIPTION
**Pull request by bileschi**
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-layers/102)

### CODE REVIEW

**Comment by davidsoergel** [comment]
:lgtm_strong:

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved.

---

*[src/models.ts, line 503 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8noTL16vTINfG9jw7q:-L8noTL16vTINfG9jw7r:b7hed27) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/models.ts#L503)):*
> ```XML
>    *   epochs: 3
>    * });
>    * console.log(h.history.loss[0]);
> ```

Careful merging here with #103

---

*[src/layers/core.ts, line 371 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8noZHj-nXjRyHXAx6a:-L8noZHj-nXjRyHXAx6b:b-qyuif6) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/layers/core.ts#L371)):*
> noteably
notably

---

*[src/layers/core.ts, line 373 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8np3S6BRxT5DXTOF3c:-L8np3S6BRxT5DXTOF3d:b-gghp74) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/layers/core.ts#L373)):*
> , for instance,
delete

---

*[src/layers/core.ts, line 374 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8np6CLBTdPcwDVOxzb:-L8np6CLBTdPcwDVOxzc:b-gd7kyi) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/layers/core.ts#L374)):*
>  for instance:
Capitalize 'For'

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-:-L8npDnlC2MCiKu-3Tvi:b-jbsf8z)*

File: src/models.ts [hunk 1]
```diff
@@ -471,10 +496,11 @@ export class Sequential extends Model {
    *   layers: [tf.layers.dense({units: 1, inputShape: [10]})]
    * });
    * model.compile({optimizer: 'sgd', loss: 'meanSquaredError'});
-   * const history = await model.fit(tf.ones([8, 10]), tf.ones([8, 1]), {
+   * const h = await model.fit(tf.ones([8, 10]), tf.ones([8, 1]), {
```
Comment (ericdnielsen) [inline src/models.ts]: Why 'h'?  seems like a bad variable choice name, and if its to deal with the h.history.loss in the lines later, it makes we think we have the wrong return type on fit?

**Comment by bileschi** [comment]
Review status: 0 of 2 files reviewed at latest revision, 8 unresolved discussions, some commit checks failed.

---

*[src/models.ts, line 171 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8o3FvzCY2r9EhUvbSf:-L8rQxqsBmPN7Ww3ZUnL:b-896fix) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/models.ts#L171)):*
<details><summary><i>Previously, nsthorat (Nikhil Thorat) wrote…</i></summary><blockquote>

await this.
</blockquote></details>

Done.

---

*[src/models.ts, line 207 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8o3IaMB__J3cMQvcvM:-L8rbLbe1UYOXxWUBnun:b-896fix) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/models.ts#L207)):*
<details><summary><i>Previously, nsthorat (Nikhil Thorat) wrote…</i></summary><blockquote>

put a comment saying this will be random
</blockquote></details>

Done.

---

*[src/models.ts, line 499 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8nZVFpX8yIgQnOB3-T-r1-499:-L8rbP8g1GoJBaQ2djih:b-8s0llx) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/models.ts#L499)):*
<details><summary><i>Previously, ericdnielsen wrote…</i></summary><blockquote>

Why 'h'?  seems like a bad variable choice name, and if its to deal with the h.history.loss in the lines later, it makes we think we have the wrong return type on fit?
</blockquote></details>

In the past I've seen the variable named `history` interact unhappily with the browser's *own* variable named `history`.  Testing ...  I guess it works ok now?  Fixed.

---

*[src/models.ts, line 503 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8noTL16vTINfG9jw7q:-L8rbt6cERL5PjAhMMbq:b-7m2y43) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/models.ts#L503)):*
<details><summary><i>Previously, davidsoergel (David Soergel) wrote…</i></summary><blockquote>

Careful merging here with #103
</blockquote></details>

noted

---

*[src/layers/core.ts, line 371 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8noZHj-nXjRyHXAx6a:-L8rc-aE0lNzcqhz-rxk:b-896fix) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/layers/core.ts#L371)):*
<details><summary><i>Previously, davidsoergel (David Soergel) wrote…</i></summary><blockquote>

> noteably
notably
</blockquote></details>

Done.

---

*[src/layers/core.ts, line 373 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8np3S6BRxT5DXTOF3c:-L8rc5hJ2xjMuozJg9GF:b-896fix) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/layers/core.ts#L373)):*
<details><summary><i>Previously, davidsoergel (David Soergel) wrote…</i></summary><blockquote>

> , for instance,
delete
</blockquote></details>

Done.

---

*[src/layers/core.ts, line 374 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8np6CLBTdPcwDVOxzb:-L8rc7b70dhg2q1J_zXj:b-896fix) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/layers/core.ts#L374)):*
<details><summary><i>Previously, davidsoergel (David Soergel) wrote…</i></summary><blockquote>

>  for instance:
Capitalize 'For'
</blockquote></details>

Done.

---

*[src/layers/core.ts, line 386 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-L8o3ZeQ8mYbbBI3Kqa5:-L8rcKhJ9mKIcflq7kVo:b-896fix) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/8e31fe8c36b989f062d1f4db516bab95ca2a6302/src/layers/core.ts#L386)):*
<details><summary><i>Previously, nsthorat (Nikhil Thorat) wrote…</i></summary><blockquote>

let's break this at a better place

```
tf.model({
    inputs: input,
    outputs: [denseOutput, activationOutput]
});
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102)*
<!-- Sent from Reviewable.io -->

**Comment by bileschi** [comment]
addressed reviewer comments.  These comments are also helpful in building the micro snippet dev tool.

---

Review status: 0 of 2 files reviewed at latest revision, 8 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/102#-:-L8rciD571wGLh5YDaLM:bd5ejtd)*
